qiskit/result/utils.py

```diff
@@ -118,10 +127,54 @@ def _adjust_creg_sizes(creg_sizes, indices):
     return new_creg_sizes
 
 
+def marginal_distribution(
+    counts: dict, indices: Optional[List[int]] = None, format_marginal: bool = False
+) -> Dict[str, int]:
+    """Marginalize counts from an experiment over some indices of interest.
```

Comment:
Is the word `marginalize` sufficiently standard to be understood or do we need to define what is meant by marginalize?

Reply:
I think probably adding a definition is a good idea